code/src/Components/TaskItem.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { todos } from '../Reducers/todos.js'
+import styled from 'styled-components'
+
+
+export const TaskItem = ({ itemIndex }) => {
+  
+  const item = useSelector((store) => store.todos.list.items[itemIndex])
```

Comment:
Since you're passing in the `item` as a prop to this component, you needn't select it from redux again. You could just pick up the item prop in your props destructuring up in the component definition:

```
export const TaskItem = ({ item }) => {
```